src/editor.css

```diff
@@ -178,8 +178,37 @@
   margin: 10px 0;
 }
 
+.template {
+  flex: 1;
+  opacity: 0.9;
+  z-index: 0;
+  transition: opacity 0.2s ease, transform 0.2s ease;
+}
+
+.template:hover {
+  transform:scale(1.05);
+  opacity: 1;
+  z-index: 1;
+  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.4);
+}
+
 .delete-file-button {
   z-index:2;
   position: absolute;
   cursor: pointer;
 }
+
+.template video,
+.template img {
+  width: 100%;
+  display: block;
+  z-index: 10;
+}
+
+.content-toolbar .choose-templates {
+  margin-left: auto;
+}
+
+.content-toolbar .selected {
+  opacity: 0.5;
```

Comment:
Hmmm, maybe set `background-color: rgba(0, 0, 0, 0.4)` instead of making it dimmer?